packages/block-editor/src/components/block-tools/index.js

```diff
@@ -159,6 +165,18 @@ export default function BlockTools( {
 			}
 			event.preventDefault();
 			expandBlock( clientId );
+		} else if ( isMatch( 'core/block-editor/group', event ) ) {
+			const clientIds = getSelectedBlockClientIds();
+			if ( clientIds.length > 1 && isGroupable( clientIds ) ) {
+				event.preventDefault();
+				const blocks = getBlocksByClientId( clientIds );
+				const groupingBlockName = getGroupingBlockName();
+				const newBlocks = switchToBlockType(
+					blocks,
+					groupingBlockName
+				);
+				replaceBlocks( clientIds, newBlocks );
+			}
```

Comment:
A spoken message when grouping is successful might be a good affordance. I tested with voiceover and there's not any indication that anything happens.

Reply:
Updated by 8cf7f34

When you run the shortcut, you should hear "Selected blocks are grouped.".

Video tested with NVDA:


https://github.com/WordPress/gutenberg/assets/54422211/7927b1b0-fd5a-4353-86a9-6d5588030270

Reply:
By the way, we may need to add a spoken message to other shortcuts as well. See #61168.